tests/components/App.test.js

```diff
@@ -12,10 +12,8 @@ import React from 'react';
 import { mount, shallow } from 'enzyme';
 import { expect } from 'chai';
 import sinon from 'sinon';
-import App from '../../app/js/components/App';
+import App from '../../app/js/components/App.jsx';
 import Header from '../../app/js/components/common/Header';
```

Comment:
Ditto.

Reply:
The file is a .js file so there's no need to add the extension. I added the extensions for all .jsx files

Reply:
Ditto